mayland-blocks/assets/alignments.css

```diff
@@ -0,0 +1,78 @@
+/*
+ * Alignments, loaded in the front-end only.
+ */
+
+body {
+	margin: 0;
+}
+
+* {
+	box-sizing: border-box;
+}
+
+.wp-site-blocks {
+	padding: 0 var(--wp--custom--margin--horizontal);
+}
+
+.wp-site-blocks > *:not(.wp-block-post-content),
+.wp-site-blocks .wp-block-post-content > * {
+	max-width: var(--wp--custom--width--default);
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.wp-site-blocks .alignwide {
+	width: var(--wp--custom--width--wide);
+	max-width: 100%;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.wp-site-blocks .alignfull {
+	transform: translateX(calc(0px - var(--wp--custom--width-horizontal)));
+	width: calc(100% + (2 * var(--wp--custom--width-horizontal)));
+	max-width: calc(100% + (2 * var(--wp--custom--width-horizontal)));
```

Comment:
I believe all of these should be `--wp--custom--margin--horizontal`

Reply:
oh yes, should we get the version from TT1 blocks? I remember we amended a few things back there

Reply:
That didn't work, so I just updated this